### ISSUE TITLE
Status post takes 30s (metatext client) (response headers)

### ISSUE THREAD

**Issue by alexbbrown** [hachyderm/community#114]
Hi folks

I'm trying to debug an issue in the metatext client that means I experience a apps 30s delay most of the time posting or replying status

does this sound familiar? 

Is there some mechanism which might cause this deliberately or accidentally in the server side?

**Comment by alexbbrown** [comment]
https://github.com/metabolist/metatext/issues/253

**Comment by alexbbrown** [comment]
Here's an example of the problematic response:
```
<NSHTTPURLResponse: 0x280552d20> { URL: https://hachyderm.io/api/v1/statuses } { Status Code: 200, Headers {
    "Cache-Control" =     (
        "no-store"
    );
    Connection =     (
        "keep-alive"
    );
    "Content-Encoding" =     (
        gzip
    );
    "Content-Security-Policy" =     (
        "base-uri 'none'; default-src 'none'; frame-ancestors 'none'; font-src 'self' https://hachyderm.io; img-src 'self' https: data: blob: https://hachyderm.io; style-src 'self' https://hachyderm.io 'nonce-8DfGHGtgLPXpUYYdR3z1Qw=='; media-src 'self' https: data: https://hachyderm.io; frame-src 'self' https:; manifest-src 'self' https://hachyderm.io; connect-src 'self' data: blob: https://hachyderm.io https://hachyderm.io wss://hachyderm.io; script-src 'self' https://hachyderm.io 'wasm-unsafe-eval'; child-src 'self' blob: https://hachyderm.io; worker-src 'self' blob: https://hachyderm.io"
    );
    "Content-Type" =     (
        "application/json; charset=utf-8"
    );
    Date =     (
        "Tue, 29 Nov 2022 05:20:21 GMT"
    );
    Etag =     (
        "W/\"a72d7e1680ff41de99f3a5567ced8078\""
    );
    "Permissions-Policy" =     (
        "interest-cohort=()"
    );
    Server =     (
        Mastodon
    );
    "Strict-Transport-Security" =     (
        "max-age=63072000; includeSubDomains",
        "max-age=31536000",
        "max-age=31536000"
    );
    "Transfer-Encoding" =     (
        Identity
    );
    Vary =     (
        "Accept-Encoding",
        "Accept, Origin"
    );
    "X-Content-Type-Options" =     (
        nosniff
    );
    "X-Frame-Options" =     (
        DENY
    );
    "X-Host" =     (
        slappy
    );
    "X-RateLimit-Limit" =     (
        300
    );
    "X-RateLimit-Remaining" =     (
        160
    );
    "X-RateLimit-Reset" =     (
        "2022-11-29T05:20:00.021544Z"
    );
    "X-Request-Id" =     (
        "ebc3dbbe-ae0a-4738-8f58-c13744825e23"
    );
    "X-Runtime" =     (
        "30.400930"
    );
    "X-XSS-Protection" =     (
        0
    );
} }
```

**Comment by quintessence** [comment]
👋 Hachyderm Mod here

FYI we were / are continuing to recover from an outage, see comment on closed issue here:

https://github.com/hachyderm/community/issues/117#issuecomment-1330033935

TLDR:

There were some issues with images and posts queuing that should be clearing up (starting at around 12:00 / noon GMT and lasting several hours due to the volume of data involved).

We'll be able to expand on this more in a public facing announcement, I mostly wanted to confirm that 1 ) we saw this and 2 ) it is tied to another issue we worked on :)
